xfrmnetlink/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "xfrmnetlink"
+version = "0.1.0"
+authors = ["Scott Zuk <zooknotic@proton.me>"]
+edition = "2018"
+
+homepage = "https://github.com/zooknotic/netlink-packet-xfrm"
+keywords = ["netlink", "ip", "linux", "ipsec"]
+license = "MIT"
+readme = "../README.md"
+repository = "https://github.com/zooknotic/netlink-packet-xfrm"
+description = "manipulate linux IPsec tunnels via netlink"
+
+[features]
+test_as_root = []
+default = ["tokio_socket"]
+tokio_socket = ["netlink-proto/tokio_socket", "tokio"]
+smol_socket = ["netlink-proto/smol_socket", "async-global-executor"]
+
+[dependencies]
+futures = "0.3.11"
+log = "0.4.8"
+thiserror = "1"
+netlink-packet-core = { path = "../netlink-packet-core" }
```

Comment:
I feel that a `version` field is meeded.

Reply:
Yes, I missed that. I also removed two dependencies that weren't being used (I originally copied the Cargo.toml from rtnetlink).